static/app/components/modals/inviteMembersModal/inviteMembersFooter.tsx

```diff
@@ -0,0 +1,80 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import ButtonBar from 'sentry/components/buttonBar';
+import InviteButton from 'sentry/components/modals/inviteMembersModal/inviteButton';
+import {useInviteMembersContext} from 'sentry/components/modals/inviteMembersModal/inviteMembersContext';
+import InviteStatusMessage from 'sentry/components/modals/inviteMembersModal/inviteStatusMessage';
+import {space} from 'sentry/styles/space';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface Props {
+  canSend: boolean;
+}
+
+export default function InviteMembersFooter({canSend}: Props) {
+  const organization = useOrganization();
+  const {
+    complete,
+    inviteStatus,
+    setInviteStatus,
+    invites,
+    pendingInvites,
+    sendInvites,
+    sendingInvites,
+    willInvite,
+  } = useInviteMembersContext();
+  const isValidInvites = invites.length > 0;
+
+  const removeSentInvites = () => {
+    const emails = Object.keys(inviteStatus);
+    let newInviteStatus = {};
+    emails.forEach(email => {
+      if (pendingInvites.emails.has(email)) {
+        newInviteStatus = {...newInviteStatus, [email]: inviteStatus[email]};
+      }
+    });
+    setInviteStatus(newInviteStatus);
+  };
+
+  return (
+    <FooterContent>
+      <div>
+        {/* TODO(mia): remove these props and use InviteMemberContext once old modal is removed */}
+        <InviteStatusMessage
+          data-test-id="invite-status-message"
+          complete={complete}
+          hasDuplicateEmails={false}
+          inviteStatus={inviteStatus}
+          sendingInvites={sendingInvites}
+          willInvite={willInvite}
+        />
+      </div>
+      <ButtonBar gap={1}>
```

Comment:
I don't think you need this button bar. There's only one button in here haha.

`ButtonBar` good for when there are multiple